src/CMakeLists.txt

```diff
@@ -160,6 +160,7 @@ endif()
 include(Docopt)
 include(mdspan)
 include(nlohmann_json)
+# include(HighFive)
```

Comment:
I see we're also checking in `src/cmake/HighFive.cmake` - should this be uncommented then? Or should we not check in `src/cmake/HighFive.cmake`?